### PR TITLE
New Package: imagemagick6 6.9.10-80

### PR DESCRIPTION
**Pull request by a-schaefers**
## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/libmagick6/build [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+
+./configure \
+    --prefix=/usr \
+    --sysconfdir=/etc \
+    --with-modules
+
+sed -i -e 's/ -shared / -Wl,-O1,--as-needed\0/g' libtool
```
Comment (dylanaraps) [inline community/libmagick6/build]: Reason for this change?

File: community/libmagick6/build [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+make DESTDIR="$1" install
```
Comment (dylanaraps) [inline community/libmagick6/build]: Missing standalone `make`?

File: community/libmagick6/depends [hunk 3]
```diff
@@ -0,0 +1,8 @@
+make
```
Comment (dylanaraps) [inline community/libmagick6/depends]: `make` can be removed.

File: community/libmagick6/depends [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+fontconfig
+libtool
```
Comment (dylanaraps) [inline community/libmagick6/depends]: `libtool make`.

**Comment by dylanaraps** [comment]
Should this package not be called `imagemagick`?

File: community/libmagick6/build (hunk 1 above)
Comment (a-schaefers) [reply community/libmagick6/build]: monkey read arch pkgbuild monkey do. 

imagemagick was complex to build. 

maybe we don't need this section after all?

Comment (a-schaefers) [reply community/libmagick6/build]: How about we remove and see how long it lasts before we need it

**Comment by a-schaefers** [comment]
now look at things

File: community/libmagick6/build (hunk 1 above)
Comment (dylanaraps) [reply community/libmagick6/build]: My `imagemagick7` package in my personal repository is a simple `./configure; make; make install`. Nothing special needed. I say remove it.

Another question, why not ImageMagick7 (the latest version)?

Comment (a-schaefers) [reply community/libmagick6/build]: I need imagemagick6 for Emacs, and it is still maintained upstream so it's fine

**Pull request merged**